### PR TITLE
create user

### PR DESCRIPTION
**Pull request by aarturoc**
This PR containst the the explanation of create user UC

[preview diagram](https://github.com/GalileiLearning/glowing-octo-dollop/tree/UC-UploadEmails/8)

### CODE REVIEW

File: 8/README.md [hunk 1]
```diff
@@ -1 +1,35 @@
...
+#### Failure
+
+1. The input information is not correct.
+2. The user already exist.
+
+### Flow Diagram
+
+```mermaid
+flowchart TD
+    A[Start] --> B{The infomration is correct?}
```
Comment (andymotagl) [inline 8/README.md:26]: nit: typo on information

File: 8/README.md [hunk 2]
```diff
@@ -1 +1,35 @@
...
+
+### Scenarios
+
+#### Success
+
+Admin add the new user.
+
+#### Failure
+
+1. The input information is not correct.
```
Comment (andymotagl) [inline 8/README.md:19]: Technically speaking information is the corporate email, isn't it? And should we check if the domain of the email being added is the same that the admin?

**Review by andymotagl — COMMENTED** [review]
I love this! So simple and straightforward.

Like so much you added the precondition section. 

Left some comments.

File: 8/README.md (hunk 2 above)
Comment (aarturoc) [reply 8/README.md:19]: I'm not sure about the second question, that dependent of the process, as I understand admin will be some one of galilei uploading the emails at least that was the last thing Mateo tall me, but if we look at the idea of having an interface for corporative user, it's good to have it in mind

**Comment by aarturoc** [comment]
i will make a new PR with the changes

**Pull request closed without merging**